code/controllers/master.dm

```diff
@@ -268,7 +268,7 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 
 	if(sleep_offline_after_initializations)
 		world.sleep_offline = TRUE
-	sleep(1)
+	sleep(1 TICKS)
```

Comment:
this changes behavior. Not sure if it should or not but it does.
1 tick with our tick lag is 0.5 seconds. sleep(1) is therefor 2 ticks.

Reply:
@MrStonedOne advise ya boomer

Reply:
This sleep is correctly changed. the goal is to make byond sleep because if nobody is connected it won't wake up until somebody connects, the time is irrelevant other then 0 will sometimes not sleep so it can't be that.